.github/ISSUE_TEMPLATE/intent-to-ship--i2s-.md

```diff
@@ -51,6 +51,47 @@ Provide instructions for how to demo your feature such as a link to a demo page.
 Add any other information that may be relevant in determining if your feature can ship.
 -->
 
+## Build checklist
+
+This section describes considerations the implementor should keep in mind while building this technical solution.
+
+- [ ] Tests written
```

Comment:
For parallelism, it would probably be good to have a comment for this line describing in more detail what tests written is calling for.